modules/creating-project-scoped-resources-for-your-project.adoc

```diff
@@ -0,0 +1,123 @@
+:_module-type: PROCEDURE
+
+[id="creating-project-scoped-resources-for-your-project_{context}"]
+= Creating project-scoped resources for your project
+
+[role='_abstract']
+Users can access _global resources_ in all {productname-short} projects, but can access _project-scoped resources_ within the specified project only.
+
+A project member who is not a cluster administrator can create the following types of project-scoped resources for their {productname-short} project:
+
+* Workbench images
+* Model-serving runtimes for KServe
+////
+When RHOAIENG-26758 is fixed, uncomment this section, after verifying that hardware profiles and accelerator profiles are both still supported
+* Hardware profiles
+* Accelerator profiles
+////
+
+All resource names must be unique within a project.
```

Comment:
Should this be `namespace` instead of `project`? Otherwise `project/namespace`. By all means, if `project` is used throughout the documentation we should stick to `project`.